source/SAMRAI/geom/CartesianCellDoubleConservativeLinearRefine.C

```diff
@@ -266,27 +307,106 @@ CartesianCellDoubleConservativeLinearRefine::refine(
          const double deltax1 = (static_cast<double>(ir1)+0.5)*fdx1-cdx1*0.5;
          const double deltax0 = (static_cast<double>(ir0)+0.5)*fdx0-cdx0*0.5;
 
-         fine_array(j,k) = coarse_array(ic0,ic1) + slope0(ic0, ic1)*deltax0 + slope1(ic0,ic1)*deltax1;
```

Comment:
Is there a need to do this in two steps with the declaration of fine_tmp instead of

`fine_array(j,k) = coarse_array(ic0,ic1)  ... `?